Update WAI-CooP (#1096)
w3c/wai-website: pages/about/projects/wai-coop/index.md

{% include_cached box.html type="start" h="2" title="Introduction" class="full" %}
{:/}

> This page provides information on the Web Accessibility Initiative Communities of Practice **(WAI-CooP) Project**, a European Commission (EC) co-funded project, Horizon 2020 Program (101004794).

Reviewer: ```suggestion
This page provides information on the Web Accessibility Initiative - Communities of Practice **(WAI-CooP) Project**, a European Commission (EC) co-funded project, Horizon 2020 Program (101004794).
```

This is the formal name of the Project.

{:#results}

**[List of Courses on Digital Accessibility](https://www.w3.org/WAI/courses/list/)**
:  Included developing a user interface with filtering functionality, creating a taxonomy to describe courses and certification on digital accessibility with mapping to the [WAI Curricula](https://www.w3.org/WAI/curricula/), promoting and gathering list entries, and creating a [Course submission form](https://www.w3.org/WAI/courses/submission/).

**[List of Web Accessibility Evaluation Tools](https://www.w3.org/WAI/test-evaluate/tools/list/)**
:  Included updating the user interface with filtering functionality, reviewing which tools' features to capture, promoting and gathering list entries, and creating a [Tool submission form](https://www.w3.org/WAI/test-evaluate/tools/submit-a-tool/).

**[List of Web Accessibility Laws and Policies](https://www.w3.org/WAI/policies/)**
:  Made extensive updates to existing policies and added new information for 22 additional countries.

> **[List of Accessibility Conformance Testing (ACT) Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/)**

Author: @iadawn - The link is to a page that says 'This page contains list of ACT Rules ...' and it consists only of lists. So I assume we can leave this change as is?

:  Included promoting, tracking, and validating implementations of “Accessibility Conformance Testing (ACT) Rules” in automated testing tools and manual testing methodologies. Also included a completed [redesign of the user interface](https://www.w3.org/WAI/standards-guidelines/act/implementations/) to better integrate implementation reports submitted by different vendors.

**Analysis of Support and Gaps in Technologies**
:  Carried out an extensive literature review to identify relevant accessibility related gaps in digital technologies, which helped to identify topics to explore further as part of the Research Symposia.

**Research Symposia on Digital Accessibility**

> : Organised and ran three online research symposia based on topics identified in previous analyses:

Author: Fixed in PR https://github.com/w3c/wai-website/pull/1417

  * [Shape the Future: Research and Development Questions in Digital Accessibility](https://www.w3.org/WAI/about/projects/wai-coop/symposium1/)
  * [Artificial Intelligence (AI) and Accessibility](https://www.w3.org/WAI/research/ai2023/)
  * [Evaluating Accessibility: Meeting Key Challenges](https://www.w3.org/WAI/about/projects/wai-coop/symposium3/)

**Regular Open Meetings**
: Organised and ran six open meetings to exchange best practices and discuss specific topics around implementing and monitoring accessibility.
  * [WAI-CooP introductory meeting](https://www.edf-feph.org/events-slug/wai-coop-first-open-meeting/)

>   * [Implementing the EU Web Accessibility Directive](https://www.edf-feph.org/events-slug/wai-coop-open-meeting2/)

Reviewer 2: ```suggestion
* [Implementation of the EU Web Accessibility Directive](https://www.edf-feph.org/events-slug/wai-coop-open-meeting2/)
```

This needs to be the original text as that is what the meeting was about.

Author: Fixed in PR https://github.com/w3c/wai-website/pull/1417

  * [Monitoring the EU Web Accessibility Directive](https://www.edf-feph.org/events-slug/wai-coop-online-meeting/)

>   * [Exploring what can be done to improve supply of and access to digital accessibility expertise](https://www.edf-feph.org/events-slug/wai-coop-open-meeting-11-october-2022/)

Reviewer 2: ```suggestion
  * [What can be done to improve supply of and access to digital accessibility expertise](https://www.edf-feph.org/events-slug/wai-coop-open-meeting-11-october-2022/)
```

Reverting to original since this was the title of the meeting.

Author: Fixed in PR #1417

  * [Exploring the feedback mechanism and complaints procedure of the Web Accessibility Directive](https://www.edf-feph.org/events-slug/wai-coop-open-meeting-7-march-2023/)

>   * [Reviewing WAI CooP activities and the future of accessibility](https://www.edf-feph.org/events-slug/wai-coop-open-meeting-14-november-2023/)

Reviewer 2: ```suggestion
  * [Review of WAI CooP activities and the future of accessibility](https://www.edf-feph.org/events-slug/wai-coop-open-meeting-14-november-2023/)
```

Reverting to the original text since that was what the meeting title was.

**[Web Accessibility Directive: Frequently Asked Questions](https://web-directive.eu/)**

> :  Provided responses to frequently asked questions (FAQ) on implementing the EU Web Accessibility Directive (WAD).

Author: @iadawn - In that case, I think it would be good to say 'Continues to provide responses to frequently asked questions (FAQ) on implementing the EU Web Accessibility Directive (WAD).'

> :  Provided responses to frequently asked questions (FAQ) on implementing the EU Web Accessibility Directive (WAD).

Reviewer 2: The original text was correct and includes the important information regarding contacting the help desk.

> :  Provided responses to frequently asked questions (FAQ) on implementing the EU Web Accessibility Directive (WAD).

Author: Fixed in PR https://github.com/w3c/wai-website/pull/1417

**[Web Content Accessibility Guidelines Evaluation Methodology (WCAG-EM) Report Tool](https://www.w3.org/WAI/eval/report-tool/)**
:  Updated functionality, improved user interface, improved translation options, and updated to include new WCAG 2.1 and 2.2 success criteria.

**[Easy Checks &mdash; A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/easy-checks/)**

> :  Updated the resource to simplify the presentation of each check, reduce the reliance on third-party tools, and improve the readability of the content. The aim was to make it easier to use as a learning resource for people new to accessibility, and to create a set of tools to perform some basic accessibility checks.

Reviewer 2: "This made it easier to use as a learning resource..." might better reflect the action that resulted rather than using 'aim' which does not highlight that the action was successful

Author: @iadawn - that sounds good. So it would be:
Updated the resource to simplify the presentation of each check, reduce the reliance on third-party tools, and improve the readability of the content. This made it easier to use as a learning resource for people new to accessibility, and to create a set of tools to perform some basic accessibility checks.

## Participation
{:#participate}

Most of the project work listed above was developed through or in coordination with the:

-   [Accessibility Education and Outreach Working Group (EOWG)](/about/groups/eowg/)
-   [Accessibility Conformance Testing (ACT) Task Force](/about/groups/task-forces/conformance-testing/)
-   [Research Questions Task Force (RQTF)](/about/groups/task-forces/research-questions/)

## About the Project
{:#about}

> WAI-CooP, Communities of Practice, was a Coordination and Support Action project, co-funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 101004794). The project started on 1 January 2021 and was completed in December 2023. WAI-CooP supported implementation of the international standards for digital accessibility through:

Reviewer: I'm pretty sure this is correct. Will publish with it and we can correct later if needed.
